x/gamm/client/cli/query.go

```diff
@@ -325,6 +326,16 @@ Example:
 	)
 }
 
+// GetConcentratedPoolIdLinkFromCFMMRequest returns concentrated pool id that is linked to the given cfmm pool id.
+func GetConcentratedPoolIdLinkFromCFMMRequest() (*osmocli.QueryDescriptor, *types.QueryConcentratedPoolIdLinkFromCFMMRequest) {
+	return &osmocli.QueryDescriptor{
+		Use:   "cl-pool-link-from-cfmm-pool-id [poolID]",
```

Comment:
nit: this feels a little verbose – thoughts on removing the `-pool-id` suffix from both of these queries?

Reply:
Agreed. I will apply the change to speed things up

Reply:
Done, please merge when CI is passing